ENGAGEHF/Managers/MessageManager/Message.swift

```diff
@@ -67,5 +87,6 @@ extension Message: Decodable {
         self.isDismissible = try container.decode(Bool.self, forKey: .isDismissible)
         self.dueDate = try container.decodeISO8601DateIfPresent(forKey: .dueDate)
         self.completionDate = try container.decodeISO8601DateIfPresent(forKey: .completionDate)
+        self.processingState = nil  // Initialize as nil since it's not stored in Firebase
```

Comment:
Not a big fan of having UI state inside models that are used for encoding/decoding data... Can we instead use a wrapper type that adds these properties?

Reply:
Also did you think about what happens to this property when the snapshot listener sends a new list of messages? Then this state is gone, right? Is this intended behavior?

Reply:
Alternative idea for discussion: Could we store the processing states in a dictionary in the MessageManager? This way, we could have the state separate and independent of the snapshot updates. Let me know what you think!